tips/aggs/aggs.md

```diff
@@ -80,13 +84,13 @@ tag=pcap packet tcp.Port
 
 ![](port-table.png)
 
-However, if we're doing full packet capture, there may be *huge* quantities of packets involved, and the query could take quite a while. If we wish to include this information in a dashboard and refer to it often, it makes sense to craft a custom agg flow. We can actually base it on the same pre-built flow as before (click "Tag Aggregates Flow" under the "Sample Flows" section of the Query Studio), we just need to make a few tweaks:
+However, if we're doing full packet capture, there may be *huge* quantities of packets involved, and the query could take quite a while. If we wish to include this information in a dashboard and refer to it often, it makes sense to craft a custom agg flow. We can actually base it on the same pre-built flow as before (select the 3-dot menu on the Tag Aggregates flow & click "Duplicate", but be sure to disable the flow until you've modified it!), we just need to make a few tweaks:
```

Comment:
Duplicating should automatically disable scheduling on the duplicate copy. I feel like this should be re-worded to reflect that. Maybe something like "but don't enable the flow until you've modified it."